### DISCUSSION TITLE
Next ID when you edit a measurement

### DISCUSSION THREAD

**Discussion by arnaud-lxbg** [dbarzin/deming#80] [Q&A]
When editing a measurement, there is, at the end, a field "Next ID" with possible value 1 to 16.
I don't find any reference in the documentation, what is this about?

**Comment by dbarzin** [comment]
Editing a control allows you to resolve input errors in controls, cancel a control, or make changes that should not be made. This function is only accessible to Deming administrators.

Controls are chained in the database on the basis of their id, and the next_id field points to the next control in the list. This field can be used to modify the next control in a control list.

This was developed during the transition from ISO 27001-2013 to ISO 27001-2022, to enable control chains to be maintained while changing the referential.
